packages/e2e-tests/specs/adding-blocks.test.js

```diff
@@ -6,11 +6,13 @@ import {
 	insertBlock,
 	getEditedPostContent,
 	pressKeyTimes,
+	switchToEditMode,
 } from '@wordpress/e2e-test-utils';
 
 describe( 'adding blocks', () => {
 	beforeEach( async () => {
 		await createNewPost();
+		await switchToEditMode();
```

Comment:
Should we just make this a global thing in `beforeEach` ? Otherwise, when would we want this to linger between test cases? If the mode isn't persisted, how could it possibly be that we need to switch to edit mode after `createNewPost` ?

Reply:
In a lot of situation we "hit" enter for example to create a block at the beginning of tests. This only works in "Edit" mode and by default we're in navigation mode.

Making it global: I thought about it. The thing is that sometimes you want to be in navigation mode as well. I guess edit is more common so it might mike sense to switch the default in tests.